internal/authorize_code_storage.go

```diff
@@ -1,11 +1,10 @@
 // Automatically generated by MockGen. DO NOT EDIT!
-// Source: github.com/ory-am/fosite/handler/oauth2 (interfaces: AuthorizeCodeStorage)
+// Source: github.com/ory/fosite/handler/oauth2 (interfaces: AuthorizeCodeStorage)
 
 package internal
 
 import (
 	context "context"
-
```

Comment:
thats correct i'm more used to run go fmt. I updated the files with goimports and added the command to the generate-mocks.sh file